scanner/src/main/kotlin/Main.kt

```diff
@@ -179,23 +182,39 @@ object Main {
             }
         }
 
-        configFile?.let {
-            ScanResultsCache.configure(it.mapper().readTree(it))
+        val config = configFile?.let {
+            require(it.isFile) {
+                "Provided configuration file is not a file: ${it.invariantSeparatorsPath}"
```

Comment:
"file is not a file" is a bit weird. How about "Provided path for configuration does not refer to a file"?

Reply:
Done.